app/kubernetes-ingress-controller/reference/feature-gates.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Feature Gates
+
+description: |
+  Learn how to customize {{ site.kic_product_name }}'s behavior using feature flags
+
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Reference
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+
+Feature gates enables contributors to add and manag`e new (and potentially experimental) functionality to the KIC in a controlled manner. The features will be "hidden" until generally available (GA) and the progress and maturity of features on their path to GA will be documented. Feature gates also create a clear path for deprecating features.
+
+Upstream [Kubernetes](https://kubernetes.io) includes [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/), which enable or disable features with flags and track the maturity of a feature using [feature stages](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
+The same definitions of `feature gates` and `feature stages` from upstream Kubernetes are used to define KIC's list of features.
+
+* [Kubernetes](https://kubernetes.io)
+* [Kubernetes feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
+* [Feature stages](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages)
```

Comment:
```suggestion
```
These should really be in the related_resources, especially since they're just repeats of links in the previous paragraph.